File: 1_semester/programmering_1/bluej/Customer/Customer.java

/**
 * Write a description of class Customer here.
 *
 * @author (your name)
 * @version (a version number or a date)
 */
public class Customer {
    // instance variables - replace the example below with your own
    private String name;
    private String email;
    private String address;
    private int yearOfBirth;

    /**
     * Constructor for objects of class Customer
     */
    public Customer(String customerName, String customerEmail) {
        name = customerName;
        email = customerEmail;
    }
    
    public String toString() {
        return "Name: " + name + "\nEmail: " + email;
    }
}

File: 1_semester/programmering_1/ruby_performance_test/customer.rb
class Customer
  def initialize(name, email, address, yearOfBirth)
    @name = name;
    @email = email;
    @address = address;
    @yearOfBirth = yearOfBirth;
  end

  def hello
    puts "Name: #{ @name }"
    nil
  end
end
